src/main/java/emissary/output/roller/journal/JournaledChannelPool.java

```diff
@@ -28,7 +28,7 @@ public class JournaledChannelPool implements AutoCloseable {
     final int max;
     final Path directory;
     final String key;
-    private final Queue<JournaledChannel> free = new LinkedList<>();
+    private final List<JournaledChannel> free = new ArrayList<>();
```

Comment:
```suggestion
    private final Deque<JournaledChannel> free = new ArrayDeque<>();
```

then you can revert the other changes in this file